pkg/scripts/bigchaindb-monit-config

```diff
@@ -161,9 +178,17 @@ check process bigchaindb
 check process tendermint
     with pidfile ${monit_pid_path}/tendermint.pid
     start program "${monit_script_path} start_tendermint ${monit_pid_path}/tendermint.pid ${monit_log_path} ${monit_log_path}"
-    restart program "${monit_script_path} start_bigchaindb ${monit_pid_path}/bigchaindb.pid ${monit_log_path} ${monit_log_path}"
+    restart program "${monit_script_path} start_tendermint ${monit_pid_path}/tendermint.pid ${monit_log_path} ${monit_log_path}"
```

Comment:
should a restart first stop and then start?

Reply:
Not necessarily, restart just happens when the service is not running.